lib/timber/integrations/error_logger.ex

```diff
@@ -0,0 +1,352 @@
+defmodule Timber.Integrations.ErrorLogger do
+  @moduledoc """
+  Handles error reports from the `:error_logger` application.
+
+  Timber can automatically log the exceptions that occur in your
+  application as exception events with all the necessary metadata
+  by registering as an `:error_logger` handler. To activate Timber's
+  error logging system, you just need to add a few configuration
+  lines:
+
+  ```
+  # Enable Timber's error capturing system
+  config :timber, :capture_errors, true
+
+  # Disable Elixir's default error capturing system
+  config :logger, :handle_otp_reports, false
+  ```
+
+  ## Elixir Logger's OTP Report Handler
+
+  The `Logger` application (which is distributed with Elixir) has an OTP report
+  handler which logs errors and will activate it by default. However, the logs
+  it writes only contain textual information without any metadata. Keeping this handler
+  active will cause duplicate errors to be reported to the log, which is why
+  we recommend disabling it using the following configuration option:
+
+  ```
+  config :logger, :handle_otp_reports, false
+  ```
+
+  However, the OTP report handler handles additional report types as well. If you
+  find that you would like these reports to be logged, just be aware that every
+  exception will be displayed twice in the log.
```

Comment:
I think you should add a note here saying that, with the default handler the report will also be broken out across multiple lines; whereas the Timber handler consolidates the information into a single line